{function: throw new Exception('Hello')}